packages/react-dom/src/__tests__/ReactDOMFizzServer-test.js

```diff
@@ -983,7 +983,6 @@ describe('ReactDOMFizzServer', () => {
   });
 
   // @gate enableSuspenseList
-
```

Comment:
Left over?

Reply:
Yeah I guess, but shouldn't we get rid of the extra line? seems like the gate comment is usually just above the test